Update portuguese.xml

TwistedJ opened the pull request:
* added translations for some words
* re-translated words - some words were not translated to what they should like: docks >docas (wrong translation) instead of docks>inicialização rápida (right translation)

and by the way, why the creators/updaters before me created the XML file with ALMOST all words with a higher case at the beginning?

TwistedJ commented:
the first archive is outdated. it has issues, commited by me...
sorry for the inconveninece.

Rikk commented:
These `&amp;` are used to create menu shortcuts using keyboard letters.

Rikk commented on PowerEditor/installer/nativeLang/portuguese.xml:
```diff
@@ -104,16 +104,16 @@
+                    <Item id="42021" name="Reproduzir"/>
                     <Item id="42022" name="Alternar Comentário de Linha"/>
-                    <Item id="42023" name="Bloquear Comentar"/>
+                    <Item id="42023" name="Bloquear Comentario"/>
```
Coment**á**rio

TwistedJ replied:
Rikk, não estou a perceber afinal o que faz o &amp; porque experimentei com as duas linguas>brasileiro e portugues, a brasileira com os &amp; e a PT sem eles, e funciona tudo da mesma maneira...
experimentei navegar pelo alt e utilizando as shortcuts fornecidas pelo programa e funcionou sem a respetiva tag. se calhar o autor do programa desenvolveu o software com uma linguagem mais atualizada que já nao precisa da tag, não sei, isso nao é o meu assunto...

Rikk replied:
Os menus ficaram sim sem os atalhos do teclado:
![portuguese](https://cloud.githubusercontent.com/assets/2916485/8994824/00345a7e-36e4-11e5-830a-ce11b8c81b14.gif)

Rikk commented on PowerEditor/installer/nativeLang/portuguese.xml:
```diff
@@ -123,9 +123,9 @@
-                    <Item id="42051" name="Painel Caráter"/>
+                    <Item id="42037" name="Modo de Coluna..."/>
+                    <Item id="42034" name="Editor de Coluna..."/>
+                    <Item id="42051" name="Painel Caráteres"/>
```
Car**ac**teres

TwistedJ replied:
Rikk, acho que com o novo acordo aortográfico isso mudou. dantes dizia-se carater no singlular e caracteres no plural, mas agora acho que é carateres mesmo... quanto ao acento está certo!

Rikk replied:
Ok... Então deve ser uma alteração feita apenas em Portugal, no Brasil não...

Rikk replied:
A verdade é que várias formas são aceitas:
http://www.flip.pt/Duvidas-Linguisticas/Duvida-Linguistica.aspx?DID=745
http://www.priberam.pt/dlpo/Caracteres

Rikk commented on PowerEditor/installer/nativeLang/portuguese.xml:
```diff
@@ -158,11 +158,11 @@
 					<Item id="43051" name="Remover Linhas Desmarcadas"/>
 					<Item id="43050" name="Inverter Marcador"/>
-					<Item id="43052" name="Encontrar carateres ..."/>
+					<Item id="43052" name="Encontrar carateres..."/>
```
cara**c**teres

TwistedJ replied:
hey,
I am not a programmer or any kind of work-tree related, I am a simple 
user, that likes notepad ++ and wanted a better translation. just that. 
before I edit the translation there were some missing terms, bad 
translations, etc.
I removed all the &amp; things becouse I dont think it needs it. as I 
said, I am not a programmer, and I dont even know the language used to 
write the lang.xml, but I dont think it needs the term &amp;, because I 
am using my edited xml file, and the basic things like menus, submenus, 
is working fine without the term &amp;, didn't tryed something more 
advanced yet.

Is it really necessary? I dont see the point of using the &amp; since 
without it is working well and besides, at notepad++ website they say 
that is something to split text or something, dont really remember, on 
the website the author dont says anything about &amp; being a shortcut 
and as third point, why spliting the text with the term? people can't 
put it on the end or the beginning of the line? why: h&amp;ome instead 
of home&amp; or &amp;home??? its way more complicated to read the 
text... I made a mistake at the first lang.xml file becouse of that.
-- 
João Pereira

xylographe replied:
See [access key at MSDN](https://msdn.microsoft.com/en-us/library/az5a73z1%28v=vs.110%29.aspx).

milipili commented:
So, is this PR ready ?

Rikk commented:
I think he broke all keyboard shortcuts (Alt key).

TwistedJ commented:
eu fiz o update, mas fiz a nivel local e graças ao xylographe entendi o que eram esses &amp;. o que aconteceu é que no website do notepad++ eu nao entendi o que eram, então entendi como texto a mais.

isso está corrigido na minha versão local, voltei a adcionar os &amps; quando finalmente percebi para que serviam. só ainda nao tinha dito nada porque nao tinha tempo.

dá para fazer o upload do meu arquivo local ou tem de ser feito tudo de novo?

reparei também que o meu ficheiro fica com mais atalhos que o XML brasileiro. eu apenas coloquei os que estavam dantes, por predefinição e penso que adicionei o de sair.

ora vê lá se já está como deveria?
![bcc04fd495c8fc21519fe6583f5f6172](https://cloud.githubusercontent.com/assets/13559490/9135584/203a7b0e-3d07-11e5-8cf5-2dc3d01f7e1d.png)

milipili commented:
English would be nice guys :)

TwistedJ commented:
milipili I messed up the xml file due to I didn't knew what was the &amp;
so, I am talking to rikk, to see if I corrected the problem, now that I know what are the &amp; keys, just that.
my last reply says:
I made de update, but at a local level, and thanks to xylograph I could understand what are the &amp; and what they do to the program. I thought, when I did the update to the xml file, that &amp; was some text that didnt belong to the xml file.
 this is corrected on my local xml file, as you can see through the screenshot. 
now I want to know if I can erase the past xml files, and re-upload the correct (with the key &amp;), to make it less confuse.
I noticed also that my xml file has more alt shortcuts than the brazilian version and I also added the shortcut "sair" (close), since the brazilian xml also has it.

that was what i said, but here I put more text than what I actually said on my mother language reply.

the image is my local xml file with the &amp;'s.

milipili commented:
No problem :) You can update this PR or create a new one.

Rikk commented:
Looks good.
Está OK.

donho commented:
#678 is more updated.
